### PR TITLE
Run tests in parallel by default in test container

### PR DESCRIPTION
**Pull request by tkdchen**
Signed-off-by: Chenxiong Qi <cqi@redhat.com>

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [n/a] Python type annotations added to new code
- [n/a] JSON/YAML configuration changes are updated in the relevant schema
- [n/a] Changes to metadata also update the documentation for the metadata
- [n/a] Pull request has a link to an osbs-docs PR for user documentation updates
- [n/a] New feature can be disabled from a configuration file

### CODE REVIEW

**Review by rcerven — APPROVED** [review]

File: tox.ini [hunk 1]
```diff
@@ -29,3 +29,14 @@ commands =
...
+
+[paths]
+source = atomic_reactor
+
+[coverage:report]
+skip_covered = true
+sort = Cover
+
+[pytest]
+addopts = -n=2 -ra --color=auto --html=__pytest_reports/atomic-reactor-unit-tests.html --self-contained-html
```
Comment (chmeliik) [inline tox.ini:41]: Would `-n=auto` be better? (attempts to auto-detect number of cores and use that many processes)

File: tox.ini [hunk 2]
```diff
@@ -29,3 +29,14 @@ commands =
         --non-interactive \
         --ignore-missing-imports \
         --package {posargs:"atomic_reactor"}
+
+[paths]
```
Comment (chmeliik) [inline tox.ini:33]: shouldn't this be `[coverage:paths]`?

File: tox.ini (hunk 1 above)
Comment (tkdchen) [reply tox.ini:41]: I would say it depends. From what I observed locally, `-n=auto` takes longer time to run the tests and the major incremental part is the preparation for processes and waiting the execution starts. Different environment might have different effect resulted by the `auto`. `-n=2` should have a good enough effect for local and CI as a default value.

File: tox.ini (hunk 2 above)
Comment (tkdchen) [reply tox.ini]: Removed. It's actually useless for this pr.

**Review by mkosiarc — APPROVED** [review]

**Review by chmeliik — APPROVED** [review]
Nice! :tada:

**Pull request merged**